src/client/client4.js

```diff
@@ -2641,9 +2641,9 @@ export default class Client4 {
         );
     };
 
-    getMarketplacePlugins = async () => {
+    getMarketplacePlugins = async (searchTerm) => {
```

Comment:
Should we use the term `filter` vs. `searchTerm` for consistency with the API?

Reply:
Done